docker-compose.yml

```diff
@@ -1,22 +1,25 @@
-version: '2.1'
+version: "3"
 services:
   spike-service:
+    build: .
     image: spike-service:latest
     ports:
-      - '8080:8080'
+      - "8080:8080"
```

Comment:
I see you really do hate those single quotes

Reply:
i do.